workalendar/africa/angola.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from datetime import timedelta
+from workalendar.core import WesternCalendar
+from workalendar.core import ChristianMixin
+from workalendar.registry import iso_register
+
+
+@iso_register('AO')
+class Angola(WesternCalendar, ChristianMixin):
+    name = 'Angola'
```

Comment:
Excellent idea to add the registration items! I'm working on extending the registry to all the continents, I won't have to do this one! :o)

Reply:
:+1: